di_website/spotlight/models.py

```diff
@@ -18,12 +18,17 @@ class Meta():
     country_code = models.CharField(max_length=100, help_text='e.g. UG, KE', default='')
     country_name = models.CharField(max_length=255)
     currency_code = models.CharField(max_length=100, help_text='UGX, KES', default='')
-
+    datasources_description = models.TextField(help_text='A description for data sources section', default='')
+    datasources_url_title = models.TextField(help_text='A title for data sources url link', default='')
+    datasources_url = models.URLField(help_text='A url link to data sources for download', default='')
```

Comment:
Actually, instead of these static ones, could you please create a generic solution, kinda like how we do for related pages. That way we can add as many links as we want. Should allow addition of either a page or url & a caption. I think we have a streamfield with a similar structure. `AbstractLinkBlock`

Reply:
All datasource links are now added with a streamfield